### ISSUE TITLE
fix_ark.sh: line 12: cd: too many arguments

### ISSUE THREAD

**Issue by ToeiRei** [GameServerManagers/LinuxGSM#2846]
## User Story

As a server admin, I want to start the server so that I can play ark.

## Basic info

* **Distro:** Debian 10
* **Game:** Ark Survival Evolved
* **Command:** start
* **LinuxGSM version:** v20.2.0

## Further Information

The script fix_ark.sh errors out saying
`/home/ark/lgsm/functions/fix_ark.sh: line 12: cd: too many arguments`

find findings in ${HOME}:
```
$ find . -name appworkshop_346110.acf
./Steam/steamapps/workshop/appworkshop_346110.acf
./.local/share/Steam/steamapps/workshop/appworkshop_346110.acf
```

**Comment by dgibbs64** [comment]
Thanks for raising this. The quick fix is. I will test this out myself and look at a fix for this

`rm ./Steam/steamapps/workshop/appworkshop_346110.acf`

**Comment by wildcat7534** [comment]
Hi !

Me too it's say now : /home/arkserver/lgsm/functions/fix_ark.sh: line 12: cd: too many arguments

I don't know how to correct the "fix_ark.sh" in ligne 12 ?

Thanks !

**Comment by h3o66** [comment]
~Closed
bug fix in version 20.3.0 included
